ports/nrf/boards/actinius_icarus/pins.csv

```diff
@@ -0,0 +1,32 @@
+P0,P0
+P1,P1
+P2,P2
+P3,P3
+P4,P4
+BUTTON,P5
+UART_RX,P6
+UART_RTS,P7
+SIM_SELECT,P8
+UART_TX,P9
+LED1,P10
+LED2,P11
+LED3,P12
+P13,P13
```

Comment:
From the specification of the board, it looks like P13 is not routed. Is this the case? If so, this line could be removed. The other pins seems to all have a purpose.

Reply:
P13 is used as A0 on the board, connected to the BAT_SENSE (voltage divider for reading the battery voltage). Should I denote this use in the pins.csv?

Reply:
What about giving it an A0 board alias name? So that it shows up as Pin.board.A0. To hint it has a special purpose.

Another alternative is to just remove that line in the `pins.csv`. It will prevent someone configuring it as general purpose IO pin, as it is not available in the Pin module to be instantiated. With the current implementation of the ADC class, it is using the AINx number as index, not the pin number. ADC(0) would mean, AIN0 aka P13. This means that reading the AIN0 value would still work without having it in the Pin class.

Reply:
I'm sorry. I'll have to rephrase a bit on my last proposal of removing `P13` from the `pins.csv`. I looked a bit closer, and ADC is suppose to also take Pin type as input. I noticed some issues that needs to be fixed. (Pin generation not populating adc_num and channel for a Pin for nrf9160).

IMO it would be nice to have the board alias names A0-A6, as it is represented on the silkscreen of the board.

Reply:
Thank you very much for the feedback! I have updated the pin aliases